pkgs/tools/video/swfmill/default.nix

```diff
@@ -13,6 +13,7 @@ stdenv.mkDerivation rec {
 
   # Fixes build with GCC 6
   env.NIX_CFLAGS_COMPILE = "-std=c++03";
+  env.NIX_LDFLAGS = "-lz";
```

Comment:
Needs a comment that it can be removed once updated past 0.3.5

Reply:
Fixed.